liquibase-standard/src/main/java/liquibase/statement/ExecutablePreparedStatementBase.java

```diff
@@ -121,6 +129,18 @@ public void execute(PreparedStatementFactory factory) throws DatabaseException {
         }
     }
 
+    protected String applyVisitors(String sql, List<SqlVisitor> sqlVisitors) {
```

Comment:
Should this method be private so we don't need to worry about overrides? As I verified the extensions and none of them is extending this class (only mssql extension makes reference to this class but not by extending it).

Reply:
The downside of having it private is that any subclass that overrides the non-final method `liquibase.statement.ExecutablePreparedStatementBase#execute` would have to reimplement the this method.